scripts/lsp-generate-bindings.el

```diff
@@ -0,0 +1,80 @@
+;;; lsp-generate-bindings.el --- Generates lsp-mode bindings  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2020  Ivan Yonchovski
+
+;; Author: Ivan Yonchovski <yyoncho@gmail.com>
+;; Keywords: convenience
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;;
+
+;;; Code:
+
+
+
+
+
+(with-temp-buffer
+  (insert-file-contents-literally "generated.protocol.schema.json")
+
+  (let ((defs (plist-get (json-parse-string (buffer-string)
```

Comment:
nit: This will only work in Emacs with native json.

Reply:
Maybe we should move the `lsp--read-json` macro from `lsp-mode` to here, or on the future move this wrapper functions like `lsp--json-serialize` to a helper/utils `.el`.

Reply:
Oh, this is a script for bindings generator, I think we will use this automatically, right? not by end-users, so I think is not a big deal use `json-parse-string` here.

Reply:
Yep, I converted it to use emacs 26 json parser.